Library/Homebrew/linkage_checker.rb

```diff
@@ -33,6 +34,7 @@ def display_normal_output
     display_items "Broken dependencies", @broken_deps
     display_items "Undeclared dependencies with linkage", @undeclared_deps
     display_items "Dependencies with no linkage", @unnecessary_deps
+    display_items "Unwanted system libraries", @unwanted_system_dylibs
```

Comment:
Can this get not shown on macOS if it's empty? Up to you what happens on Linux in the empty case.